module/checker.go

```diff
@@ -11,9 +11,20 @@ import (
 	"github.com/golang/protobuf/ptypes"
 	"github.com/golang/protobuf/ptypes/duration"
 	"github.com/golang/protobuf/ptypes/timestamp"
-	"github.com/lyft/protoc-gen-star"
+	pgs "github.com/lyft/protoc-gen-star"
 )
 
+var unknown = ""
+var httpHeaderName = "^:?[0-9a-zA-Z!#$%&'*+-.^_|~\x60]+$"
+var httpHeaderValue = "^[^\x00-\x08\x0A-\x1F\x7F]"
```

Comment:
Should this match the entire string? I think it just matches the first character.